addons/l10n_in/static/src/helpers/hsn_summary.js

```diff
@@ -20,7 +21,7 @@ patch(accountTaxHelpers, {
             const quantity = base_line.quantity;
             const product_values = base_line.product_values;
             const uom = base_line.uom || {};
-            const taxes_data = base_line.taxes_data;
+            const taxes_data = base_line.taxes;
```

Comment:
By consistency with the python code, it's not supposed to be `taxes` but `taxes_data`:
https://github.com/odoo/odoo/blob/31cbbfda4df700227526a2b97fe299dc802fb665/addons/l10n_in/models/account_tax.py#L49